server/.env.example

```diff
@@ -1,7 +1,12 @@
-BROKER_USERNAME=macondorivers1demo
-BROKER_PASSWORD=
-BROKER_API_KEY=
-BROKER_ACCOUNT_ID=
-BROKER_BASE_URL=https://demo-api.ig.com/gateway/deal
+IG_USERNAME=macondorivers1demo
+IG_PASSWORD=
+IG_API_KEY=
+IG_ACCOUNT_ID=
+IG_BASE_URL=https://demo-api.ig.com/gateway/deal
```

Comment:
This isn't exactly necessary but I thought I may as well rename this now for a new multi broker world